src/routes/safe/components/Transactions/TxList/TokenTransferAmount.tsx

```diff
@@ -34,7 +34,7 @@ export const TokenTransferAmount = ({ assetInfo }: TokenTransferAmountProps): Re
         height={26}
         onError={(error) => {
           error.currentTarget.onerror = null
-          error.currentTarget.src = assetInfo.tokenType === TokenType.ERC721 ? NFTIcon : TokenPlaceholder
+          error.currentTarget.src = assetInfo.tokenType === TransactionTokenType.ERC721 ? NFTIcon : TokenPlaceholder
```

Comment:
I would say that also this could be a TokenType instead a TransactionTokenType, because I understand it also comes from an asset, but I couldn't ensure because I'm not doing a deep check on the code